ds-prep-python-variables/main.py

```diff
@@ -0,0 +1,36 @@
+first_name = 'James'
+last_name = "Nakano"
+full_name = first_name + " " + last_name
+print(full_name)
+
+height_in_inches = 69
+print(height_in_inches, type(height_in_inches))
+height_in_inches_float = float(height_in_inches)
+print(height_in_inches_float, type(height_in_inches_float))
+height_in_meters = 2.54 * height_in_inches_float/100
+print(height_in_meters)
+
+eats_plants = True
+eats_animals = False
+if eats_plants == True or eats_animals == True:
+  is_animal = True
+else:
+  is_animal = False
+if eats_plants == True and eats_animals == True:
+  is_omnivore = True
+else:
+  is_omnivore = False
```

Comment:
Try assigning the `is_animal` and `is_omnivore` values without using `if-else` conditionals